src/resources/UsageAnalytics/Read/Exports/tests/Exports.spec.ts

```diff
@@ -100,6 +115,19 @@ describe('Exports', () => {
         });
     });
 
+    describe('generateVisitExportWithBody', () => {
+        it('makes a POST call to the specific Exports url with a request body', () => {
+            const params: GenerateVisitExportWithBodyParams = {
+                from: '987',
+                to: '654',
+            };
+            exports.generateVisitExportWithBody(params);
+
+            expect(api.post).toHaveBeenCalledTimes(1);
+            expect(api.post).toHaveBeenCalledWith(`${Exports.baseUrl}/visits/create`, params, undefined);
+        });
```

Comment:
```suggestion
        it('makes a POST call to the specific Exports url with a request body', async () => {
            const params: GenerateVisitExportWithBodyParams = {
                from: '987',
                to: '654',
            };
            await exports.generateVisitExportWithBody(params);

            expect(api.post).toHaveBeenCalledTimes(1);
            expect(api.post).toHaveBeenCalledWith(`${Exports.baseUrl}/visits/create`, params, undefined);
        });
```

Reply:
@gdostie any particular benefits to this change? None of our tests in this file (or in this other PR I put up here- https://github.com/coveo/platform-client/pull/860) have async/await, I can add them to everything here if you'd prefer.

Reply:
You can read the benefits about it here https://typescript-eslint.io/rules/no-floating-promises/. This rule is part of the typescript-eslint recommended rules config. Its just that we aren't using the recommended config at the moment (soon we will be)

Reply:
In my experience, these often generate leaks during test executions which induces flakiness. When unawaited, the promise resolves or errors when the test already has teardown and is executing another test of the suite

Reply:
TIL, thanks. I'll add them to all the tests in this file then, I'll wait until the eslint rules kick in to enforce it for all our other tests.

I was thinking of adding DPF as codeowners for the resources we own here so we can get added to PRs on changes to them in this repo.

Reply:
Perfect thanks! Don't worry about existing tests, I already went over them in a local branch. My comment is mostly to avoid the work of also fixing new tests that are coming in.